packages/fx-core/src/common/azure-hosting/azureOps.ts

```diff
@@ -51,7 +53,8 @@ export class AzureOperations {
   public static async zipDeployPackage(
     zipDeployEndpoint: string,
     zipBuffer: Buffer,
-    config: AzureUploadConfig
+    config: AzureUploadConfig,
```

Comment:
I am thinking if we need to add logger into some utility class? The alternative way is leverage Error to pass some information, otherwise it force unit test to take care of logger?

Reply:
The logger is an optional parameter. Does unit test need to take care of it?